src/main/groovy/com/moowork/gradle/node/NodeExtension.groovy

```diff
@@ -33,6 +33,8 @@ class NodeExtension
 
     boolean download = false
 
+    boolean taskRules = true
```

Comment:
I think we might need a more descriptive name for this, but right now I can't think of a good one

Reply:
i went back and forth about it, it is a gradle plugin so in theory shouldn't be abstracting gradle concepts, task rules is the concept being configured. Could be 'dynamic task generation' or something but that is re-labelling something that already has a name.. a task rule 🤷‍♂️

Reply:
added some doco to solve for descriptiveness =)